Aztec/Classes/Libxml2/Converters/Out/OutHTMLConverter.swift

```diff
@@ -1,55 +1,215 @@
 import Foundation
 import libxml2
 
+
+// MARK: - HTML Prettifier!
+//
 extension Libxml2.Out {
     class HTMLConverter: Converter {
-        
-        typealias EditContext = Libxml2.EditContext
-        typealias Node = Libxml2.Node
-        typealias ElementNode = Libxml2.ElementNode
-        typealias RootNode = Libxml2.RootNode
-        
-        required init() {
-        }
 
-        /// Converts the a Libxml2 Node into HTML representing the same data.
+        // MARK: - Typealiases
+
+        typealias Attribute         = Libxml2.Attribute
+        typealias StringAttribute   = Libxml2.StringAttribute
+        typealias ElementNode       = Libxml2.ElementNode
+        typealias Node              = Libxml2.Node
+        typealias TextNode          = Libxml2.TextNode
+        typealias CommentNode       = Libxml2.CommentNode
+        typealias RootNode          = Libxml2.RootNode
+
+        /// Indentation String to be applied
         ///
-        /// - Parameters:
-        ///     - rawNode: the Libxml2 Node to convert.
+        var indentationString = "  "
+
+        /// Indicates whether we want Pretty Print or not
         ///
-        /// - Returns: a String object representing the specified HTML data.
+        var prettyPrint = false
```

Comment:
The converters were designed to be transient, and created whenever a specific conversion is needed.  Having these settings be variables makes it feel like you may want to store a single converter for separate conversions, which is not ideal (because for example this class is not thread safe).

Can we change these to constants and configure them through the required initializer instead?

PS: changing this after a conversion has started is risky as well.